src/benchmark/tables.hpp

```diff
@@ -0,0 +1,89 @@
+#include <Lightweight/DataMapper/DataMapper.hpp>
+#include <Lightweight/SqlConnection.hpp>
+#include <Lightweight/SqlDataBinder.hpp>
+#include <Lightweight/SqlQuery.hpp>
+#include <Lightweight/SqlQueryFormatter.hpp>
+#include <Lightweight/SqlScopedTraceLogger.hpp>
+#include <Lightweight/SqlStatement.hpp>
+#include <Lightweight/SqlTransaction.hpp>
+struct lists;
```

Comment:
```suggestion

struct lists;
```

just a readability thought